packages/react-jss/src/JssProvider.js

```diff
@@ -1,21 +1,28 @@
 // @flow
 import React, {Component, type Node} from 'react'
 import PropTypes from 'prop-types'
-import defaultJss, {createGenerateId, type Jss, type GenerateId, SheetsRegistry} from 'jss'
+import defaultJss, {
+  createGenerateId,
+  type Jss,
+  type GenerateId,
+  SheetsRegistry,
+  type CreateGenerateIdOptions
+} from 'jss'
 import type {Context, Managers} from './types'
 import JssContext from './JssContext'
 import memoize from './memoize-one'
 
 /* eslint-disable react/require-default-props, react/no-unused-prop-types */
 
-type Props = {
-  jss?: Jss,
-  registry?: SheetsRegistry,
-  generateId?: GenerateId,
-  classNamePrefix?: string,
-  disableStylesGeneration?: boolean,
-  media?: string,
-  children: Node
+interface Props {
```

Comment:
why do you use in this case and in case of CreateGenerateIdOptions an interface instead of type?

Reply:
I'm just being used to use interfaces because of TS

Reply:
I think even in ts interfaces have a bit different purpose than types., we should use type when possible and interface when needed.